src/Backend/Modules/Faq/Form/FaqDeleteType.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Backend\Modules\Faq\Form;
+
+use Backend\Core\Engine\Model;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+final class FaqDeleteType extends AbstractType
```

Comment:
You can always extend the form for the exceptions, or even add an extra field in the controller.
I don't think that one exception is enough to justify this overhead